src/options.ts

```diff
@@ -76,7 +76,7 @@ export async function resolveOptions(options: Partial<VitePWAOptions>, viteConfi
     globDirectory: outDirRoot,
     offlineGoogleAnalytics: false,
     cleanupOutdatedCaches: true,
-    dontCacheBustURLsMatching: /\.[a-f0-9]{8}\./,
+    dontCacheBustURLsMatching: /[.-][a-f0-9]{8}\./,
```

Comment:
I think the `.` here needs to be escaped?

Reply:
IIRC, in a character class (square brackets) any character except `^`, `-`, `]` or `\` is a literal, and `-` require escaping only if not the first nor last one.

Reply:
About the dot, IntelliJ showing a redundant escaping warning:

![imagen](https://user-images.githubusercontent.com/6311119/201535105-d54d302b-d89a-4172-adfd-f21bc6125594.png)